pkg/apis/stackconfigpolicy/v1alpha1/webhook.go

```diff
@@ -112,6 +112,12 @@ func validSettings(policy *StackConfigPolicy) field.ErrorList {
 	if policy.Spec.Elasticsearch.IndexTemplates.ComposableIndexTemplates != nil {
 		settingsCount += len(policy.Spec.Elasticsearch.IndexTemplates.ComposableIndexTemplates.Data)
 	}
+	if policy.Spec.Elasticsearch.Config != nil {
+		settingsCount += len(policy.Spec.Elasticsearch.Config.Data)
+	}
+	if policy.Spec.Elasticsearch.SecretMounts != nil {
```

Comment:
Do we want to check when a mount point is reused?

```yaml
  elasticsearch:
    secretMounts:
      - secretName: my-secret
        mountPath: /tmp/foo
      - secretName: my-secret2
        mountPath: /tmp/foo
```

Reply:
Good point!!